zeebe-job-worker/src/main/kotlin/job/multiinstance/ShipWithInsurance.kt

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package job.multiinstance
+
+import io.camunda.zeebe.client.ZeebeClient
+import io.camunda.zeebe.client.ZeebeClientBuilder
+import io.camunda.zeebe.client.api.response.ActivatedJob
+import io.camunda.zeebe.client.api.worker.JobClient
+import io.camunda.zeebe.client.api.worker.JobHandler
+import java.time.Duration
+import java.util.Scanner
+
+object ShipWithInsurance {
+    @JvmStatic
+    fun main(args: Array<String>) {
```

Comment:
I'm still of the mind that you should just have one main function for the project and the register all the workers there, simplifying startup a ton. I don't see much reason to not start them all at once? Or what's the case where you would only want to start a single worker but not the others? This would also remove the need to add any run configurations